rest/src/main/kotlin/builder/message/create/MessageCreateBuilder.kt

```diff
@@ -86,6 +107,35 @@ public sealed interface MessageCreateBuilder {
      */
     public suspend fun addFile(path: Path): NamedFile =
         addFile(path.fileName.toString(), ChannelProvider { path.readChannel() })
+
+}
+
+internal fun buildMessageFlags(
+    base: MessageFlags?,
+    suppressEmbeds: Boolean? = null,
```

Comment:
```suggestion
    suppressEmbeds: Boolean?,
```
this parameter is passed to every call of this function, i think it shouldn't have a default value